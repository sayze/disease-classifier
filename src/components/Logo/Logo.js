import React from 'react'
import { SvgIcon, Typography, withStyles } from '@material-ui/core'
import PropTypes from 'prop-types'
import { ReactComponent as BrandIcon } from './icon.svg'

const styles = theme => ({
  root: {
    display: 'flex',
    alignItems: 'center',
    color: theme.palette.common.white,
  },
  text: {
    marginLeft: theme.spacing(1),
    fontFamily: 'Chalkboard',
  },
})

const Logo = ({ classes }) => (
  <div className={classes.root}>
    <SvgIcon component={BrandIcon} viewBox="0 0 600 476.6" />
    <Typography className={classes.text} variant="h6">
      Disease Classifier
    </Typography>
  </div>
)

Logo.propTypes = {
  classes: PropTypes.objectOf(PropTypes.any).isRequired,
}

export default withStyles(styles)(Logo)
