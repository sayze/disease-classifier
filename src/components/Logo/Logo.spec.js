import React from 'react'
import { mount } from 'enzyme'
import Logo from '.'

describe('Logo', () => {
  it('should render default Logo', () => {
    const component = mount(<Logo />)
    expect(component).toMatchSnapshot()
  })
})
