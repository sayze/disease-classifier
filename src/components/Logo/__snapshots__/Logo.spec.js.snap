// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render default Logo 1`] = `
<WithStyles(Logo)>
  <Logo
    classes={
      Object {
        "root": "Logo-root-1",
        "text": "Logo-text-2",
      }
    }
  >
    <div
      className="Logo-root-1"
    >
      <WithStyles(ForwardRef(SvgIcon))
        component={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          }
        }
        viewBox="0 0 600 476.6"
      >
        <ForwardRef(SvgIcon)
          classes={
            Object {
              "colorAction": "MuiSvgIcon-colorAction",
              "colorDisabled": "MuiSvgIcon-colorDisabled",
              "colorError": "MuiSvgIcon-colorError",
              "colorPrimary": "MuiSvgIcon-colorPrimary",
              "colorSecondary": "MuiSvgIcon-colorSecondary",
              "fontSizeInherit": "MuiSvgIcon-fontSizeInherit",
              "fontSizeLarge": "MuiSvgIcon-fontSizeLarge",
              "fontSizeSmall": "MuiSvgIcon-fontSizeSmall",
              "root": "MuiSvgIcon-root",
            }
          }
          component={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            }
          }
          viewBox="0 0 600 476.6"
        >
          <ForwardRef(SvgIcon)
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 600 476.6"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 600 476.6"
            >
              icon.svg
            </svg>
          </ForwardRef(SvgIcon)>
        </ForwardRef(SvgIcon)>
      </WithStyles(ForwardRef(SvgIcon))>
      <WithStyles(ForwardRef(Typography))
        className="Logo-text-2"
        variant="h6"
      >
        <ForwardRef(Typography)
          className="Logo-text-2"
          classes={
            Object {
              "alignCenter": "MuiTypography-alignCenter",
              "alignJustify": "MuiTypography-alignJustify",
              "alignLeft": "MuiTypography-alignLeft",
              "alignRight": "MuiTypography-alignRight",
              "body1": "MuiTypography-body1",
              "body2": "MuiTypography-body2",
              "button": "MuiTypography-button",
              "caption": "MuiTypography-caption",
              "colorError": "MuiTypography-colorError",
              "colorInherit": "MuiTypography-colorInherit",
              "colorPrimary": "MuiTypography-colorPrimary",
              "colorSecondary": "MuiTypography-colorSecondary",
              "colorTextPrimary": "MuiTypography-colorTextPrimary",
              "colorTextSecondary": "MuiTypography-colorTextSecondary",
              "displayBlock": "MuiTypography-displayBlock",
              "displayInline": "MuiTypography-displayInline",
              "gutterBottom": "MuiTypography-gutterBottom",
              "h1": "MuiTypography-h1",
              "h2": "MuiTypography-h2",
              "h3": "MuiTypography-h3",
              "h4": "MuiTypography-h4",
              "h5": "MuiTypography-h5",
              "h6": "MuiTypography-h6",
              "noWrap": "MuiTypography-noWrap",
              "overline": "MuiTypography-overline",
              "paragraph": "MuiTypography-paragraph",
              "root": "MuiTypography-root",
              "srOnly": "MuiTypography-srOnly",
              "subtitle1": "MuiTypography-subtitle1",
              "subtitle2": "MuiTypography-subtitle2",
            }
          }
          variant="h6"
        >
          <h6
            className="MuiTypography-root Logo-text-2 MuiTypography-h6"
          >
            Disease Classifier
          </h6>
        </ForwardRef(Typography)>
      </WithStyles(ForwardRef(Typography))>
    </div>
  </Logo>
</WithStyles(Logo)>
`;
